test/assets/import/invalid-package/api/create-service.json

```
@@ -0,0 +1,3 @@
+{
+    "serviceName": "{{ .context.service }}"
```

We are using `[[` and `]]` for templateing in files to avoid clashing with the templating synatax of other services (like webhook-service)

---

This is an example of an invalid package, so the wrong syntax was added by design.

---

Hmm I would prefer to have just some bogus data in there that's easily distinguished from real data.

---

So you would like the formatting to be correct and the data to be non-existing (wrong mappings, etc.)—any suggestions from data that is easily distinguished from valid data?

---

You could use something like this:
```suggestion
    "serviceName": "{{ .context.service.not-templated }}"
```

---

Just added the changes